Os/Baremetal/SystemResources.cpp

```diff
@@ -0,0 +1,41 @@
+// ======================================================================
+// \title  Baremetal/SystemResources.cpp
+// \author mstarch
+// \brief  hpp file for SystemResources component implementation class
+//
+// \copyright
+// Copyright 2021, by the California Institute of Technology.
+// ALL RIGHTS RESERVED.  United States Government Sponsorship
+// acknowledged.
+//
+// ======================================================================
+
+namespace Os {
+
+SystemResources::SystemResourcesStatus SystemResources::getCpuCount(U32& cpuCount) {
+    // Assumes 1 CPU
+    cpuCount = 1;
+    return SYSTEM_RESOURCES_OK;
+}
+
+SystemResources::SystemResourcesStatus SystemResources::getCpuTicks(CpuTicks& cpu_ticks, U32 cpu_index) {
+    // Always 100 percent
+    cpu_ticks.used = 1;
+    cpu_ticks.total = 1;
+    return SYSTEM_RESOURCES_OK;
+}
+
+
+SystemResources::SystemResourcesStatus SystemResources::getMemUtil(MemUtil& memory_util) {
```

Comment:
Does this function violate the rule against dynamic allocation after initialization? Also, it seems to rely on behavior of the memory allocator that may not be true. The allocation could occur anywhere that is free, not necessarily at the free space boundary.

Reply:
This implementation doesn't work as expected anyway.  This code can be removed.